js/front-panel.js

```diff
@@ -230,7 +230,8 @@ class FrontPanel {
           const [fromIdx, toIdx] = [connection.from, connection.to].sort((a, b) => b - a)
           const from = this.connectors[fromIdx];
           const to = this.connectors[toIdx];
-          return from.name + " -> " + to.name;
+          const color = connection.color;
+return `${from.name}<span style="color: ${color}"> -> </span>${to.name}`;
```

Comment:
Woops, indentation gone wrong, must be because I forgot to do that. Still getting used to doing Github reviews. :smile:

Regarding that arrow, I think the long-rightwards-arrow ⟶ looks quite nice here, what do you think? Wondering, would it give problems on less mainstream devices/browsers/printers?
```suggestion
          return `${from.name}<span style="color: ${color}"> &#10230; </span>${to.name}`;
```